### ISSUE TITLE
Nationzone doesn't work properly

### ISSUE THREAD

**Issue by Eryk1983S** [TownyAdvanced/Towny#3305]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. Enable the nationzones in the config
2. Create a nation
3. As a player, who doesn't belong to that nation - come to the chunk nearby of a town of that nation, where the nationzone should be
4. Destroy a block

### What is the expected output?
1. You shouldn't be able to destroy a block there, because only the nation-members should be allowed to build/destroy in the nationzone

### What is your Towny version?
0.93.0.0

### What is your spigot/craftbukkit version?
Spigot 1.12.2-R0.1-SNAPSHOT

### Please use Pastebin.com to link the following files:
Error: https://pastebin.com/HBWKaa7t

**Comment by LlmDl** [comment]
This might already be fixed:
```
0.93.1.0:
    - Fix for nationZones affecting nation-less towns when the only_capitals setting is false.
```

Please test using Towny 0.93.1.0 ( https://cdn.discordapp.com/attachments/321710042091094026/583426209988476946/Towny.jar ) See changelog 0.93.0.1 and forward for changes: https://git.io/vxiWf

**Comment by Eryk1983S** [comment]
First of all: Thank you very much for your fast answer :)

I've updated Towny to 0.93.1.0., as you've recommended, but the error still appears in the console and the foreigner is still able to grief in the nationzone. It's undependend from the setting "only_capitals" - earlier I had "false", then I switched to "true" and it doesn't change something at all.

Here is the pastebin: https://pastebin.com/Btd3qWZC
For the first look it's the same, but now the version of Towny is updated.

I don't understand Java, but my humble experience with servers tells me, that the plugin can be uncompatible with 1.12.2. (in such cases there's always weird error, that contains the version of the engine). Later I'll try to test with 1.13. for your information, but I please very much to fix it with 1.12.

**Comment by Eryk1983S** [comment]
Sorry for double-posting, but I want, that you notice this message. I realized, that I've pasted only the half of the error (I thought, it's a repeat)...
Here's the whole error, after destroying one block (vines) in the nationzone: https://pastebin.com/mWND1i2m
And here my config-file: https://pastebin.com/6xYFNY8e
The "townyperms" are default.

**Comment by LlmDl** [comment]
If you use 0.93.0.0 and have the capitals_only option set to true does it still show errors/still allow players to break things in what should be a nationzone?

**Comment by LlmDl** [comment]
I did just test 0.93.1.0 PR 5 on a 1.14.3 server and everything was behaving like normal. Doesn't help you on 1.12 very much though.

**Comment by Eryk1983S** [comment]
Meanwhile I've deleted the plugin from the server and uploaded it again.
The configuration files I've set manually. Simple: Started the whole
installation from the beginning.

I can remember, that first the errors never appeared again on the console,
but the foreigner was still able to destroy in the nationzone.

Now, after I received a message about your last post, everything seems to
work fine :) and independent from setting only_capitals. Even, when it's
set to "false".

Whatever you did, thank you very much :)

sob., 20 lip 2019 o 18:53 LlmDl <notifications@github.com> napisał(a):
